spring-kafka/src/main/java/org/springframework/kafka/config/KafkaListenerEndpointRegistrar.java

```diff
@@ -74,6 +79,25 @@ public KafkaListenerEndpointRegistry getEndpointRegistry() {
 		return this.endpointRegistry;
 	}
 
+	/**
+	 * Return the list of {@link HandlerMethodArgumentResolver}.
+	 * @return the list of {@link HandlerMethodArgumentResolver}.
+	 * @since 2.5
+	 */
+	public List<HandlerMethodArgumentResolver> getCustomMethodArgumentResolvers() {
+		return this.customMethodArgumentResolvers;
+	}
+
+	/**
+	 * Add custom methods arguments resolvers to {@link KafkaListenerAnnotationBeanPostProcessor}
+	 * Default empty list.
+	 * @param methodArgumentResolvers the list of {@link HandlerMethodArgumentResolver} to use.
+	 * @since 2.5
```

Comment:
I think your change is fully non-braking, so it's OK to have this feature in the current `2.4.2` version.